docs/FAQs.md

```diff
@@ -33,4 +42,73 @@ If the issue persists, navigate to the folder's properties and choose the option
 5. In the Advanced Security Settings, select the option `Replace all child object permission entries with inheritable permission entries from this object`.
 6. Apply the changes and confirm.
 
-</details>
+## `Initialize-NatNetwork` issues
+
+<!-- 
+#####################################################################
+#################### Could not import HNS module ####################
+#####################################################################
+-->
+### 1. Could not import HNS module
+
+Ensure 'Hyper-V Host Compute Service' Windows features are enabled.
+
+```PowerShell
+Get-WindowsOptionalFeature -Online | `
+    Where-Object { $_.FeatureName -match "Microsoft-Hyper-V(-All)?$" } | `
+    Select-Object FeatureName, Possible, State, RestartNeeded
+```
+
+To enable a feature:
+
+```PowerShell
+Enable-WindowsOptionalFeature -Online -FeatureName '<Feature-Name-Here>' -All -NoRestart
+
+# Restart device to apply changes
+# Restart-Computer
+```
+
+[Restart the operating system](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.management/restart-computer?view=powershell-7.4) to apply changes.
+
+<!-- 
+#####################################################################
+############### New-HNSNetwork command does not exist ###############
+#####################################################################
+-->
+### 2. New-HNSNetwork command does not exist
+
+The built-in `HostNetworkingService` PowerShell module does not include the `New-HNSNetwork` command. This command is available through third-party modules or scripts that are **NOT** managed or signed by Microsoft.
+
+> [!CAUTION]  
+> _Microsoft does **NOT** take any responsibility for issues that may arise from installing and/or execution of commands in these third-party modules/scripts. Proceed at your own risk._
+
+You can install the _third-party_ HNS module from the PowerShell Gallery or directly from the Kubernetes repository.
+
+**Option 1:** Install from the PowerShell Gallery:
+
+    > [!TIP]  
+    > If `HostNetworkingService` module exists, the -AllowClobber flag is necessary due to a naming conflict with the built-in `HostNetworkingService` module commands.
+
+    ```powershell
+    Install-Module -Name HNS -Scope CurrentUser
```

Comment:
I'd suggest that we drop `HNS` altogether and just go with the one on Microsoft's SDN repo (which will actually be first party but not 3rd party). With that, we can just install it without the warning.

HNS authorship is not very clear, and that could be a security risk.

```
PS C:\Users\Administrator> (Get-Module HNS).Author
nwood
```

Reply:
Could you share the link for _"the one on Microsoft's SDN repo"_?

Reply:
It's the one you have on line 102 below `$Uri = 'https://raw.githubusercontent.com/microsoft/SDN/dd4e8708ed184b49d3fddd611b6027f1755c6edb/Kubernetes/windows/hns.psm1'`

Reply:
https://github.com/microsoft/SDN/blob/master/Kubernetes/windows/hns.psm1